google-platforms-case-study.md

```diff
@@ -0,0 +1,185 @@
+
+GA provide insights in the form of dimensions and metrics in their reports, i.e. Very simply put, GA will answer the question: How many users clicked the primary button on the home page? By using the primary button as a dimension and number of clicks as the metric.
+
+### GA4 provide five main categories
+
+- Acquisition reports
+- Engagement reports
+- Monetization reports
+- Demographics reports
+- Tech reports
+
+For the sake of this study, I will cover three of these five reports. Acquisition, engagement and demographics. In addition to exploration reports.
+
+#### Acquisition reports
+
+In GA4 you will find two different types of acquisition reports.
+
+- User acquisition reports: Will help you answer how people who have never used your website or app found your website or app.
+- Traffic acquisition reports: Will help you answer what led both new and returning users to visit your website or app.
+  (Analytics Help, 2023)
+
+These reports will provide great insights into how people found your website and/or what made them visit.
+examples of channels or sources people could have arrived from and triggered them to visit could be:
+
+- Paid Social ads, like ads on facebook or instagram.
+- Paid search, like paid ads on google.
+- Organic search (no paid), like in the search result of google.
+- Direct traffic, people who just enter your website in the browser and go there.
+
+#### Engagement reports
+
+In GA4 you will find four different types of engagement reports
+
+- Events, could be anything happening or user action on the page. Like: page view, button click, form submit, scrolled.
+- Conversions, is an event, but usually an event which is view as the final step in a number of events. Like: making a purchase or complete registration.
+- Pages and screens, lets you view on which page the engagement is happening. How many views a certain page has, total events triggered on a certain page or how many users visited the certain page.
+- Landing pages, gives you an overview of the engagement on the different landing pages (the first page of the website people viewed when entering the website)
+
+#### Demographics reports
+
+Lets you see from what city or country your traffic/users are coming from.
+
+#### Exploration reports
+
+Makes you customize your own reports, which could provide greater insights to different user journeys.
+
+In GA4 you could setup a lott of complex reports, which might fit your needs better insights then whats provided by the standard reports.
+Lets say you wanted to view a report with the following conditions: Users who made a purchase, but only users who clicked the primary button on the home page and which did not arrive from paid ads. You could add all these filters each time on one of the standard reports, but if this is a report you would like to view more frequently, to make an exploration report for it would be much more efficient.
```

Comment:
```suggestion
Let's say you wanted to view a report with the following conditions: Users who made a purchase but only users who clicked the primary button on the home page and did not arrive from paid ads. You could add all these filters each time on one of the standard reports, but if this is a report you would like to view more frequently, to make an exploration report for it would be much more efficient.
```